next/components/forms/info-components/StatusBar.tsx

```diff
@@ -0,0 +1,55 @@
+/** A react component with state provided through context
+ * which shows a red status bar with white text on top of the page
+ */
+import cx from 'classnames'
+import React, { createContext, useContext, useState } from 'react'
+
+import ErrorIcon from '../icon-components/ErrorIcon'
+import { SectionContainer } from '../segments/SectionContainer/SectionContainer'
+
+const StatusBarContext = createContext<{
+  statusBarContent: React.ReactNode
+  setStatusBarContent: React.Dispatch<React.SetStateAction<React.ReactNode>>
+}>({
+  statusBarContent: null,
+  setStatusBarContent: () => undefined,
+})
+
+interface StatusBarProviderProps {
+  children?: React.ReactNode
+}
+
+export const StatusBarProvider: React.FC<StatusBarProviderProps> = ({ children }) => {
+  const [statusBarContent, setStatusBarContent] = useState<React.ReactNode>(null)
+  return (
+    <StatusBarContext.Provider value={{ statusBarContent, setStatusBarContent }}>
+      {children}
+    </StatusBarContext.Provider>
+  )
+}
+
+export const useStatusBarContext = () => useContext(StatusBarContext)
+
+interface StatusBarProps {
+  className?: string
+}
+
+export const StatusBar: React.FC<StatusBarProps> = ({ className }) => {
+  const { statusBarContent } = useStatusBarContext()
+  return (
+    statusBarContent && (
+      <div className={cx('w-full bg-negative-700 text-white', className)}>
+        <div className="container mx-auto h-full flex items-center justify-center">
+          <SectionContainer>
+            <div className="row flex items-center py-4">
+              <span className="hidden md:flex mr-3">
+                <ErrorIcon solid className="w-5 h-5" />
+              </span>
+              <p className="text-p2">{statusBarContent}</p>
+            </div>
+          </SectionContainer>
```

Comment:
SectionContainer styles does not looks good (px - 32px, need to be - 20px), i think we don't need that component in StatusBar just add px-5 lg:px-0 to div.
```suggestion
            <div className="row flex items-center py-4 px-5 lg:px-0">
              <span className="hidden md:flex mr-3">
                <ErrorIcon solid className="w-5 h-5" />
              </span>
              <p className="text-p2">{statusBarContent}</p>
            </div>
```

Reply:
I've tried the suggestion but I do not think that is the desired behaviour - it's a bit of a pity we don't have an example with long enough content in designs, but I'd expect the sectioncontainer px (or mx) behaviour here, instead of `px-5 lg:px-0`.

But ultimately, the plan is to show this to the product/design, and if it turns out I'm wrong I'll grab this piece of code 🙂 for now merging as it is, but thanks for pointing this out.